packages/web-app-files/src/views/Personal.vue

```diff
@@ -131,9 +131,7 @@ export default {
         })
 
         // Load quota
```

Comment:
its ok to load the quota in the background, but can you please write it as [iife](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) instead of then?